main.go

```diff
@@ -4,6 +4,38 @@ import (
 	"fmt"
 )
 
+func ElapsedDays(arg1, arg2, arg3 int) {
```

Comment:
package `api` will not work.

Reply:
can you give the arguments more descriptive names?
is `arg1` year? month? day?

Reply:
Pedro, would you be so kind to provide a docstring for this function?

Example:
```go
// ElapsedDays returns the number of days since the 1st of January of that year.
```